docs/learning-powershell/README.md

```diff
@@ -102,6 +102,15 @@ Note that all bash commands should continue working on PowerShell session.
 - [PowerShell Documentation](https://docs.microsoft.com/powershell)
 - [Interactive learning with PSKoans](https://aka.ms/pskoans)
 
+### Make PowerShell Beautiful
+
+You can use [oh-my-posh](https://ohmyposh.dev/) and see
+its source code in [JanDeDobbeleer/oh-my-posh](https://github.com/JanDeDobbeleer/oh-my-posh).
+
+An Example:
+
+![image](https://user-images.githubusercontent.com/50760269/206097676-58283ed0-d244-4c5c-813d-1218f8bed27f.png)
+
```

Comment:
Nothing against **oh-my-posh** (I use **posh-git** instead) but does this really belong here in this document. What other 3rd-party tools would want to be listed here?

Also, we already have docs about customizing the shell experience.

Reply:
> Nothing against **oh-my-posh** (I use **posh-git** instead) but does this really belong here in this document. What other 3rd-party tools would want to be listed here?
> 
> Also, we already have docs about customizing the shell experience.

Yeah it feels a little odd to single out oh-my-posh here when there are quite a few projects in the same area (https://github.com/Jaykul/PowerLine, https://github.com/dahlbyk/posh-git, https://github.com/starship/starship).

If the goal is to have a link to send to folks, I'd recommend writing up a markdown gist with your settings and links to projects

Reply:
> Also, we already have docs about customizing the shell experience.

Has the doc any links to third-party tools?

Reply:
> Nothing against **oh-my-posh** (I use **posh-git** instead) but does this really belong here in this document. What other 3rd-party tools would want to be listed here?
> 
> Also, we already have docs about customizing the shell experience.

You're right. And where are the docs about customizing the shell experience? Tnx.